docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
<p>For consistency with most of the tutorials, would not capitalize every noun (only other tutorial that does it is <code>ICNN Dual Solver </code>)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I think I disagree, capitalizing headings is also done in other notebooks. It is also the dominant style in most notebook / articles / submission guides of conferences in ML.</p>